docs/js-client-sdk.chromestorageengine._constructor_.md

```diff
@@ -0,0 +1,21 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@eppo/js-client-sdk](./js-client-sdk.md) &gt; [ChromeStorageEngine](./js-client-sdk.chromestorageengine.md) &gt; [(constructor)](./js-client-sdk.chromestorageengine._constructor_.md)
+
+## ChromeStorageEngine.(constructor)
+
+Constructs a new instance of the `ChromeStorageEngine` class
+
+**Signature:**
+
+```typescript
+constructor(storageMap: ChromeStorageAsyncMap<string>, storageKeySuffix: string);

```

Comment:
these should have been generated from a previous commit. is there a command that I should have run?

Reply:
yes you can run `yarn run docs`, but it's also created by the pre-commit script

https://github.com/Eppo-exp/js-client-sdk/blob/e2f01d438686423609b726adfc49525f68f46d47/package.json#L17

Although there's a mis-ordering of events so the commit ends up happening without including the generated docs, which is probably why they were left out. I'm increasingly inclined to removing these autogenerated docs altogether though. I don't believe anyone views them

Reply:
is it this stuff? https://eppo-exp.github.io/js-client-sdk/js-client-sdk.html